src/utils/validators/validateEmail.ts

```diff
@@ -0,0 +1 @@
+export const validateEmail = (v: string): boolean => v.match(".+@.+") !== null;
```

Comment:
I think the regex should be a little more strict this regex matches `someemail@domain` or `1@1` which arent valid emails